intelmq/lib/mixins/stomp.py

```diff
@@ -74,7 +93,11 @@ def prepare_stomp_connection(self) -> Tuple['stomp.Connection', dict]:
           `<STOMP connection>` object.
         """
         ssl_kwargs, connect_kwargs = self.__get_ssl_and_connect_kwargs()
+        # Note: here we coerce `port` to int just to be on the safe
+        # side, as some historical versions of `etc/feeds.yaml` used
+        # to set it to a string.
         host_and_ports = [(self.server, int(self.port))]
+        self.__patch_ssl_related_stomp_stuff(host_and_ports)
```

Comment:
```suggestion
        self.__patch_ssl_in_stomp(host_and_ports)
```

Or so, to make it look better ;)